lib/collections/schemas/products.js

```diff
@@ -301,6 +302,14 @@ export const ProductVariant = new SimpleSchema({
     type: Date,
     optional: true
   },
+  // TODO: REVIEW - Does this need to exist? Should we use workflow instead?
+  // Should it be called 'history' or something else instead?
+  // Should this go into the Logger instead? Is the logger robust enough for this?
```

Comment:
we use a `transactions` schema elsewhere as a general unstructured logging array.

Reply:
I've seen the `transactions` schema on billing, shipping, and discounts - is that the preferred method of logging these types of things. I think it could be useful to have a structured array of events that have occurred on products, inventory, etc - but I acknowledge this is a pretty big feature to sneak into this PR

Reply:
Would love a discussion on how best to store logs for things like this. I think theres some value in having a structured set of data for events that have happened on products, orders, inventory, users, etc and was experimenting with that. Completely open to removing it entirely in favor of something else

Reply:
Regarding this discussion, I also considered using a `history` object similar to what is found on orders here https://github.com/reactioncommerce/reaction/blob/marketplace/lib/collections/schemas/orders.js#L22-L35